corehq/apps/data_interfaces/pillow.py

```diff
@@ -85,6 +85,12 @@ def _get_associated_form(self, change):
 
     def _has_applicable_changes(self, case_updates, rule):
         action_definition = CaseDeduplicationActionDefinition.from_rule(rule)
+        if not action_definition.include_closed:
+            # If the rule shouldn't include closed cases, then a case being closed is an actionable event
+            closes_case = any(case_update.closes_case() for case_update in case_updates)
+            if closes_case:
```

Comment:
I think the intermediate variable adds clarity. It's similar to how I would call a function rather than inline a function, even if the function wasn't used elsewhere. Although the comment does add some readability, it is meant to explain the necessity for the logic, rather than the logic itself. I also view comments are more transitory